users/vieting/experiments/switchboard/ctc/feat/data.py

```diff
@@ -3,14 +3,20 @@
 from sisyphus import Path
 from i6_core import corpus as corpus_recipe
 from i6_core import text
+from i6_core.returnn.config import CodeWrapper
 from i6_core.returnn.oggzip import BlissToOggZipJob
 from i6_experiments.common.datasets.switchboard.corpus_eval import get_hub5e00
 from i6_experiments.common.setups.rasr.gmm_system import GmmSystem
 from i6_experiments.common.setups.rasr.util import OggZipHdfDataInput
 
 
 def get_corpus_data_inputs_oggzip(
-    gmm_system, partition_epoch, context_window=None, returnn_root=None, returnn_python_exe=None
+    gmm_system,
+    partition_epoch,
+    context_window=None,
+    returnn_root=None,
+    returnn_python_exe=None,
+    preprocessing=False
```

Comment:
You could make this configurable, not just boolean.
```suggestion
    pre_process=None
```